.circleci/config.yml

```diff
@@ -220,7 +220,6 @@ jobs:
           command: |
             sudo apt-get update
             sudo apt-get -y install g++
-            sudo -E script/setup-musl
```

Comment:
Please don't remove this! It's still used by the other linux builds.

Reply:
Oh, wait, my bad. This is in the build for aarch64 right. Sorry!

Reply:
It's still there for the amd64 build. I only removed it from the aarch64 build.